src/main/kotlin/no/skatteetaten/aurora/mean/genie/ApplicationConfig.kt

```diff
@@ -28,4 +51,100 @@ class ApplicationConfig : BeanPostProcessor {
             it.realmName = "MEAN-GENIE"
         }
     }
+
+    @Bean
+    fun websocketClient(
+        @Qualifier("openshift") tcpClient: TcpClient,
+        @Value("\${integrations.openshift.url}") openshiftUrl: String,
+        @Value("\${integrations.openshift.tokenLocation:file:/var/run/secrets/kubernetes.io/serviceaccount/token}") token: Resource
+    ): ReactorNettyWebSocketClient {
+        return ReactorNettyWebSocketClient(
+            HttpClient.create()
+                .baseUrl(openshiftUrl)
+                .headers {
+                    it.add(HttpHeaders.AUTHORIZATION, "Bearer ${token.readContent()}")
+                    it.add("User-Agent", "Mean-Genie")
```

Comment:
Jeg vet ikke helt hvilke regler vi har for å sette User-Agent mellom våre komponenter, men kan/bør den være knyttet opp mot ApplicationConfig.applicationName?

Reply:
endret fra "Mean-Genie" til applicationName